test/functional/plugin/shada_spec.lua

```diff
@@ -715,6 +722,7 @@ describe('In autoload/shada.vim', function()
         '  # Unexpected enum value: expected one of 0 (CHARACTERWISE), '
         .. '1 (LINEWISE), 2 (BLOCKWISE)',
         '  + rt   type         10',
+        '  + ru   is_unnamed   FALSE',
```

Comment:
Also there are register entry tests which a) contain each key value equal to default one and b) contain each key value *not* equal to default (but yet with a correct type).